src/javascript/node-index.js

```diff
@@ -127,3 +127,6 @@ function createContext (width, height, options) {
 }
 
 module.exports = createContext
+const { WebGLTexture } = require("./webgl-texture")
```

Comment:
```suggestion
const { WebGLTexture } = require('./webgl-texture')
```
and add one and only one blank line to the end of the file to placate `standard` running on AppVeyor.
* Scroll to the very bottom of https://ci.appveyor.com/project/dhritzkiv/headless-gl/builds/47591784
> standard | snazzy && tape test/*.js | faucet
standard: Use JavaScript Standard Style (https://standardjs.com)
standard: Run `standard --fix` to automatically fix some problems.
C:\projects\headless-gl\src\javascript\node-index.js
  130:34  error  Strings must use singlequote                   quotes
  132:43  error  Newline required at end of file but not found  eol-last
✖ 2 problems
Command exited with code 1